Some code in decenc_openjpeg.c is now deprecated with recent version of OpenJPEG

I'm receiving the following error when -DUSE_OpenJPEG=ON 
```
/home/runner/work/NCEPLIBS-g2c/NCEPLIBS-g2c/g2c/src/decenc_openjpeg.c: In function ‘enc_jpeg2000’:
/home/runner/work/NCEPLIBS-g2c/NCEPLIBS-g2c/g2c/src/decenc_openjpeg.c:494:5: error: ‘bpp’ is deprecated: Use prec instead [-Werror=deprecated-declarations]
  494 |     cmptparm.bpp = (OPJ_UINT32)nbits;
      |     ^~~~~~~~
In file included from /home/runner/work/NCEPLIBS-g2c/NCEPLIBS-g2c/g2c/src/decenc_openjpeg.c:21:
/usr/include/openjpeg-2.5/openjpeg.h:750:45: note: declared here
  750 |     OPJ_DEPRECATED_STRUCT_MEMBER(OPJ_UINT32 bpp, "Use prec instead");
      |                                             ^~~
/usr/include/openjpeg-2.5/openjpeg.h:80:83: note: in definition of macro ‘OPJ_DEPRECATED_STRUCT_MEMBER’
   80 | #define OPJ_DEPRECATED_STRUCT_MEMBER(memb, msg) __attribute__ ((deprecated(msg))) memb
      |                                                                                   ^~~~
cc1: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/g2c_objlib.dir/build.make:779: src/CMakeFiles/g2c_objlib.dir/decenc_openjpeg.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:1200: src/CMakeFiles/g2c_objlib.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

--- a/src/decenc_openjpeg.c
+++ b/src/decenc_openjpeg.c
@@ -491,7 +491,6 @@ enc_jpeg2000(unsigned char *cin, g2int width, g2int height, g2int nbits,
         0,
     };
     cmptparm.prec = (OPJ_UINT32)nbits;
-    //cmptparm.bpp = (OPJ_UINT32)nbits;
     cmptparm.sgnd = 0;
     cmptparm.dx = 1;
     cmptparm.dy = 1;

--- a/src/decenc_openjpeg.c
+++ b/src/decenc_openjpeg.c
@@ -491,7 +491,7 @@ enc_jpeg2000(unsigned char *cin, g2int width, g2int height, g2int nbits,
         0,
     };
     cmptparm.prec = (OPJ_UINT32)nbits;
-    cmptparm.bpp = (OPJ_UINT32)nbits;
+    //cmptparm.bpp = (OPJ_UINT32)nbits;
     cmptparm.sgnd = 0;
     cmptparm.dx = 1;
     cmptparm.dy = 1;

